types/messaging.ts

```diff
@@ -1,4 +1,54 @@
-export interface CfrMessage {
-  slug: string;
-  trigger: object;
+// TODO: This needs expanding
+interface MessageTrigger {
```

Comment:
Andrei just did a bunch of work in https://phabricator.services.mozilla.com/D76475 for trigger schemas